pootle/apps/pootle_store/models.py

```diff
@@ -1437,8 +1439,11 @@ def delete(self, *args, **kwargs):
         super(Store, self).delete(*args, **kwargs)
 
         self.clear_all_cache(parents=False, children=False)
-        if parent is not None:
-            parent.update_all_cache()
+        for p in parents:
+            p.update_all_cache()
+
+            for vfcti in vfctis:
```

Comment:
I think the point being that you have to explain what it means here.  Good luck to anyone reviewing this code at a later date.  If you where treading with fear that you'd go over 80 chars, remember that its more important that the code id readable before you worry that the line is 100 characters long.